Dockerfile

```diff
@@ -1,12 +1,27 @@
-FROM adoptopenjdk:11-jre-hotspot
-
-RUN apt-get update \
-    && apt-get install -y unzip \
-    && rm -rf /var/lib/apt/lists/*
+ARG VERSION=3.6
+ARG JAVA_VERSION=11-jre-hotspot
 
+FROM adoptopenjdk:${JAVA_VERSION}
 ARG VERSION
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+LABEL "maintainer"="DITA Open Toolkit Development Team"
+LABEL "org.opencontainers.image.authors"="Robert D Anderson, Jarno Elovirta, Roger W. Fienhold Sheen"
```

Comment:
My preference would be to not put our names here, just because none of us wants people contacting us directly.

Reply:
Taken from `https://www.dita-ot.org/who_we_are` - I'll replace with the link.